ansible/tasks/initdb.yml

```diff
@@ -61,4 +92,17 @@
     password: "{{ db_password }}"
     force_basic_auth: yes
   with_items: "{{ db.authkeys }}"
-  when: result.status == 404
+  when: dbexists is defined and dbexists.status == 404
+
+- name: secure db against anonymous access
```

Comment:
enabling secure mode by default is fine. In that case putting it in the deployment role is ok. But then why would I run a deployment playbook to change a configuration and toggle it on off when I'm not bringing down or tearing down the container? Isn't that more appropriate for a task?